### DISCUSSION TITLE
Need to get this following view

### DISCUSSION THREAD

**Discussion by aliraza-me** [livewire/livewire#4677] [1. Help]
I need to resolve the following UI with it can someone help if it's possible 
<img width="1221" alt="Screenshot 2022-02-15 at 11 04 28 PM" src="https://user-images.githubusercontent.com/32343489/154122082-bd7c767b-306b-4d7f-a2d4-edd0e1e3b456.png">

**Comment by ousid** [comment]
What's your question?
